Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -521,10 +522,11 @@ void InsertDiscCallback(u64 userdata, int cyclesLate)
 
 void ChangeDisc(const std::string& newFileName)
 {
+	bool not_cpu = !Core::IsCPUThread();
```

Comment:
I think it would be clearer to have `bool is_cpu = Core::IsCPUThread();` here and invert the other logic.